lib/src/api/v3/message.rs

```diff
@@ -386,7 +386,7 @@ impl Message {
     /// Pushes a `Recipient` to the `Recipients` field of the `Message`
     pub fn push_recipient(&mut self, recipient: Recipient) {
         if self.have_email_fields_filled() {
-            panic!(PUSHING_RECIPIENTS_WITH_RECEIVERS_ERROR_MESSAGE);
+            panic!("{}", PUSHING_RECIPIENTS_WITH_RECEIVERS_ERROR_MESSAGE);
```

Comment:
Is there any advantage on using this approach instead? With this constant we are already proving a `static` string slice which is the "argument" for this macro, I find it a bit redundant but perhaps I'm missing something?

Reply:
This is due to the following warning:
```Rust
warning: panic message is not a string literal
   --> lib/src/api/v3/message.rs:399:20
    |
399 |             panic!(PUSHING_RECIPIENTS_WITH_RECEIVERS_ERROR_MESSAGE);
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(non_fmt_panic)]` on by default
    = note: this is no longer accepted in Rust 2021
help: add a "{}" format string to Display the message
    |
399 |             panic!("{}", PUSHING_RECIPIENTS_WITH_RECEIVERS_ERROR_MESSAGE);
    |                    ^^^^^
help: or use std::panic::panic_any instead
    |
399 |             std::panic::panic_any(PUSHING_RECIPIENTS_WITH_RECEIVERS_ERROR_MESSAGE);
    |
    ```